src/MassiveAction.php

```diff
@@ -759,7 +759,7 @@ public static function getAllMassiveActions($item, $is_deleted = false, ?CommonD
             }
 
             // Specific actions
-            $actions += $item->getSpecificMassiveActions($checkitem);
+            $actions += $item->getSpecificMassiveActions($checkitem, $items_id);
```

Comment:
If you need to forbid a massive action for a given item, you can override the `getForbiddenSingleMassiveActions()` method in the target class.